lib/loader.js

```diff
@@ -0,0 +1,86 @@
+module.exports = (mcData) => {
+  const data = {
```

Comment:
maybe let's simply loop over the object ?

Reply:
yeah that would remove the need to duplicate the api, which then decrease the cost of the extra package

Reply:
updated to use Object.assign on a new object